### PR TITLE
Bug 1426158 Update documentation on creating a local role

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1426158

### CODE REVIEW

File: admin_guide/manage_authorization_policy.adoc [hunk 1]
```diff
@@ -614,10 +614,39 @@ $ oc project <project_you_want_to_add_the_local_role_exampleview_to>
 
 Optionally, annotate it with a description.
 
-To use the new role, run:
+To create policy.json:
```
Comment (barleyer) [inline admin_guide/manage_authorization_policy.adoc]: I think policybinding file should like this:
```
apiVersion: v1
kind: PolicyBinding
metadata:
  name: <string>
policyRef:
  name: <role-name>
  namespace: <project-name>
roleBindings: null
```

File: admin_guide/manage_authorization_policy.adoc [hunk 2]
```diff
@@ -614,10 +614,39 @@ $ oc project <project_you_want_to_add_the_local_role_exampleview_to>
...
+    "policyRef": {
+
+        "namespace": "<project-name>"
+
+    }
+----
+
+To create policy-binding, run:
+----
+$ oc create -f policy.json -n <project-name>
```
Comment (barleyer) [inline admin_guide/manage_authorization_policy.adoc]: Maybe using name "policybinding.yaml" is More appropriate

**Comment by bmcelvee** [comment]
Thanks @barleyer! PR updated.

File: admin_guide/manage_authorization_policy.adoc [hunk 3]
```diff
@@ -600,10 +601,10 @@ rules:
...
   - pods
   - builds
-  verbs:
-  - get
-	- list
-	- watch
+verbs:
+- get
+  - list
+  - watch
```
Comment (stuartchuan) [inline admin_guide/manage_authorization_policy.adoc]: These 4 lines need aligned with upper line.
`
apiVersion: v1
kind: Role
metadata:
  name: exampleview
rules:
- apiGroups: null
  attributeRestrictions: null
  resources:
  - pods
  - builds
  verbs:
  - get
  - list
  - watch
`

Comment (stuartchuan) [inline admin_guide/manage_authorization_policy.adoc]: These 4 lined need aligned with upper line.
`
apiVersion: v1
kind: Role
metadata:
  name: exampleview
rules:
- apiGroups: null
  attributeRestrictions: null
  resources:
  - pods
  - builds
  verbs:
  - get
  - list
  - watch
`

Comment (stuartchuan) [inline admin_guide/manage_authorization_policy.adoc]: These 4 lined need aligned with upper line.
`
apiVersion: v1
kind: Role
metadata:
  name: exampleview
rules:
- apiGroups: null
  attributeRestrictions: null
  resources:
  - pods
  - builds
  verbs:
  - get
  - list
  - watch
`

Comment (stuartchuan) [inline admin_guide/manage_authorization_policy.adoc]: These 4 lines need aligned with upper line.

**Comment by bmcelvee** [comment]
Thanks, @stuartchuan! PR updated. 

@openshift/team-documentation PTAL :sunflower:

File: admin_guide/manage_authorization_policy.adoc [hunk 4]
```diff
@@ -602,8 +603,8 @@ rules:
   - builds
   verbs:
   - get
-	- list
-	- watch
+  - list
+  - watch
 ----
 
 Then, to use the current project, run:
```
Comment (kalexand-rh) [inline admin_guide/manage_authorization_policy.adoc]: Are you changing to the project that you want to apply the role to? If so, this line isn't clear.

File: admin_guide/manage_authorization_policy.adoc [hunk 5]
```diff
@@ -614,10 +615,31 @@ $ oc project <project_you_want_to_add_the_local_role_exampleview_to>
 
 Optionally, annotate it with a description.
 
-To use the new role, run:
+To create policybinding.yaml:
```
Comment (kalexand-rh) [inline admin_guide/manage_authorization_policy.adoc]: I'd say something like, "Save the following role binding in the *_policybinding.yaml_* file:"

I might also add the `cat` command to save the file.

File: admin_guide/manage_authorization_policy.adoc [hunk 6]
```diff
@@ -614,10 +615,31 @@ $ oc project <project_you_want_to_add_the_local_role_exampleview_to>
...
+kind: PolicyBinding
+metadata:
+  name: <string>
+policyRef:
+  name: <role-name>
+  namespace: <project-name>
+roleBindings: null
+----
+
+To create policy-binding, run:
```
Comment (kalexand-rh) [inline admin_guide/manage_authorization_policy.adoc]: s/policy-binding/the PolicyBinding

**Review by kalexand-rh — COMMENTED** [review]
One question and a couple of nitpicks. This change sounds good. :)

File: admin_guide/manage_authorization_policy.adoc (hunk 4 above)
Comment (bmcelvee) [reply admin_guide/manage_authorization_policy.adoc]: I don't think so. This was existing content, and from what I understand it's adding a role to an existing project.

**Comment by bmcelvee** [comment]
Thanks, @stuartchuan and @kalexand-rh!

File: admin_guide/manage_authorization_policy.adoc (hunk 4 above)
Comment (kalexand-rh) [reply admin_guide/manage_authorization_policy.adoc]: That's fair, but it's still not super clear why it's the "current" project.

Comment (bmcelvee) [reply admin_guide/manage_authorization_policy.adoc]: I tried to clarify it a bit with the latest push :)

Comment (kalexand-rh) [reply admin_guide/manage_authorization_policy.adoc]: That sounds better. Thank you! :fireworks:

**Pull request merged**